ingest/build-configs/ncbi/bin/transform-host

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+"""
+Transforms host name from using a provided map of host names.
+"""
+import argparse
+import csv
+import json
+from sys import stderr, stdin, stdout
+
+
+def parse_args():
```

Comment:
```suggestion
def parse_args() -> argparse.Namespace:
```

Reply:
Included in https://github.com/nextstrain/avian-flu/pull/40/commits/eb5e068ef2d4260b01c7ac49fa805b539fb05e83